src/font/lv_font.c

```diff
@@ -64,7 +64,17 @@ bool lv_font_get_glyph_dsc(const lv_font_t * font_p, lv_font_glyph_dsc_t * dsc_o
                            uint32_t letter_next)
 {
     LV_ASSERT_NULL(font_p);
-    return font_p->get_glyph_dsc(font_p, dsc_out, letter, letter_next);
+    LV_ASSERT_NULL(dsc_out);
```

Comment:
What about this implementation?
```c
bool lv_font_get_glyph_dsc(const lv_font_t * font_p, lv_font_glyph_dsc_t * dsc_out, uint32_t letter,
                           uint32_t letter_next)
{
    dsc_out->resolved_font = NULL;
    const lv_font_t * f = font_p;
    while(f) {
        if (font_p->get_glyph_dsc(font_p, dsc_out, letter, letter_next)) {
            return true;
        }
        f = f->fallback;
    }
}
```

The main advantage is ` lv_font_glyph_dsc_t g = {.resolved_font = NULL};` is not required.